### PR TITLE
chore(codepipeline): make examples compile

### PR DESCRIPTION
**Pull request by kaizencc**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-codepipeline/README.md [hunk 1]
```diff
@@ -312,36 +351,40 @@ you access the appropriate property of the interface returned from `variables`,
...
 const myAction = new MyAction({
   // ...
+  actionName: 'myAction',
 });
 new OtherAction({
   // ...
   config: myAction.variables.myVariable,
+  actionName: 'otherAction',
 });
 ```
```
Comment (kaizen3031593) [inline packages/@aws-cdk/aws-codepipeline/README.md:365]: I created a fixture that basically holds the dummy `MyAction` and `OtherAction` classes. Hopefully that's the right way to go here...

Comment (rix0rrr) [reply packages/@aws-cdk/aws-codepipeline/README.md:365]: Sure! You could also have stuck them in `default` and that would have been fine too, but this is good.

**Review by rix0rrr — APPROVED** [review]

File: packages/@aws-cdk/aws-codepipeline/README.md [hunk 2]
```diff
@@ -16,14 +16,14 @@
 To construct an empty Pipeline:
 
 ```ts
-import * as codepipeline from '@aws-cdk/aws-codepipeline';
-
+// construct an empty Pipeline:
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-codepipeline/README.md:19]:
```suggestion
// Construct an empty Pipeline
```

File: packages/@aws-cdk/aws-codepipeline/README.md [hunk 3]
```diff
@@ -16,14 +16,14 @@
...
 const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline');
 ```
 
 To give the Pipeline a nice, human-readable name:
 
 ```ts
+// give the Pipeline a nice, human-readable name:
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-codepipeline/README.md:26]:
```suggestion
// Give the Pipeline a nice, human-readable name
```

File: packages/@aws-cdk/aws-codepipeline/README.md [hunk 4]
```diff
@@ -40,6 +40,7 @@ the creation of the Customer Master Keys by passing `crossAccountKeys: false`
 when defining the Pipeline:
 
 ```ts
+// disable the creation of the Customer Master Keys
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-codepipeline/README.md:43]:
```suggestion
// Don't create Customer Master Keys
```

File: packages/@aws-cdk/aws-codepipeline/README.md [hunk 5]
```diff
@@ -50,6 +51,7 @@ you can configure it by passing `enableKeyRotation: true` when creating the pipe
 Note that key rotation will incur an additional cost of **$1/month**.
 
 ```ts
+// enable key rotation for the generated KMS key
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-codepipeline/README.md:54]:
```suggestion
// Enable key rotation for the generated KMS key
```

**Review by rix0rrr — APPROVED** [review]

**Pull request merged**